bundles/org.eclipse.passage.lic.base/src/org/eclipse/passage/lic/base/io/Settings.java

```diff
@@ -66,49 +61,36 @@ public Settings(Supplier<Path> base) {
 	}
 
 	public Map<String, Object> get() throws LicensingException {
-		HunterForSettingsFiles hunter = new HunterForSettingsFiles();
-		try {
-			Files.walkFileTree(base.get(), hunter);
+		try (Stream<Path> files = settingFilesIn(base.get())) {
+			return collectProperties(files);
```

Comment:
just `properties(files)`, it a _query_ - returns result and does not alter a state, to be named after something it is querying, meaning noun.

Reply:
Done.